learning/katas/go/Core Transforms/Map/MapElements/pkg/task/task.go

```diff
@@ -0,0 +1,33 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import "github.com/apache/beam/sdks/go/pkg/beam"
+
+func ApplyTransform(s beam.Scope, input beam.PCollection) beam.PCollection {
+	processFn := &multiplyByFn{
+		Factor: 5,
+	}
+	return beam.ParDo(s, processFn, input)
+}
+
+type multiplyByFn struct {
+	Factor int
+}
+
+func (f *multiplyByFn) ProcessElement(input int) int {
```

Comment:
I'd suggest having the empty ProcessElement function revealed for the students. They can then fill in the function body.